content/en/docs/Getting-started/supply-chain-security.md

```diff
@@ -0,0 +1,295 @@
+   [kaniko-build : build-and-push] INFO[0000] Executing 0 build triggers
+   [kaniko-build : build-and-push] INFO[0000] Unpacking rootfs as cmd RUN echo "hello world" > hello.log requires it.
+   [kaniko-build : build-and-push] INFO[0000] RUN echo "hello world" > hello.log
+   [kaniko-build : build-and-push] INFO[0000] Taking snapshot of full filesystem...
+   [kaniko-build : build-and-push] INFO[0000] cmd: /bin/sh
+   [kaniko-build : build-and-push] INFO[0000] args: [-c echo "hello world" > hello.log]
+   [kaniko-build : build-and-push] INFO[0000] Running: [/bin/sh -c echo "hello world" > hello.log]
+   [kaniko-build : build-and-push] INFO[0000] Taking snapshot of full filesystem...
+   [kaniko-build : build-and-push] INFO[0000] Pushing image to 10.101.134.48/tekton-test
+   [kaniko-build : build-and-push] INFO[0001] Pushed image to 1 destinations
+
+   [kaniko-build : write-url] 10.101.134.48/tekton-test
+   ```
+
+## Retrieve and verify the artifact provenance
+
+Tekton Chains silently monitored the execution of the PipelineRun. It recorded
+and signed the provenance metadata, information about the container that the
+PipelineRun built and pushed.
+
+1. Get the PipelineRun UID:
+
+   ```bash
+   export PR_UID=$(tkn pr describe --last -o  jsonpath='{.metadata.uid}')
+   ```
+
+1. Fetch the metadata and store it in a JSON file:
+
+   ```bash
+   tkn pr describe --last \
+   -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-pipelinerun-$PR_UID}" \
+   | base64 -d > metadata.json
+   ```
+
+1. View the provenance:
+
+   ```bash
+   cat metadata.json | jq -r '.payload' | base64 -d | jq .
+   ```
+
+   The output contains a detailed description of the build:
+
+   {{% readfile file="samples/provenance.json" code="true" lang="json" %}}
+
+1. To verify that the metadata hasn't been tampered with, check the signature
+   with `cosing`:
+
+   ```bash
+   cosign verify-blob-attestation --insecure-ignore-tlog \
+   --key k8s://tekton-chains/signing-secrets --signature metadata.json \
+   --type slsaprovenance --check-claims=false /dev/null
+   ```
+
+   The output confirms that the signature is valid:
+  
+   ```
+   Verified OK
+   ```
+
+## Further reading  
+
+- [Learn about Tekton Chains and Supply Chain Security][chains-overview].
+- [Getting To SLSA Level 2 with Tekton and Tekton Chains (Google Open Source
+  Blog)][blog-post].
+- [Check more examples on the Tekton Chains repository][chains-repo].
+    
+[chains-overview]: /docs/concepts/supply-chain-security/
+[chains-repo]: https://github.com/tektoncd/chains/tree/main/examples
+[minikube]: https://minikube.sigs.k8s.io/docs/start/
+[kubectl]: https://kubernetes.io/docs/tasks/tools/#kubectl
+[tkn]: /docs/cli/
+[jq]: https://stedolan.github.io/jq/download/
+[cosign]: https://docs.sigstore.dev/cosign/installation/
+[blog-post]: https://opensource.googleblog.com/2023/03/getting-to-slsa-level-2-with-tekton-and-tekton-chains.html
```

Comment:
Could we link to this https://tekton.dev/blog/2023/04/19/getting-to-slsa-level-2-with-tekton-and-tekton-chains/ instead? The version on the tekton blog points back to the original on googleblog anyways